codalab/worker/main.py

```diff
@@ -245,6 +251,25 @@ def main():
             args.work_dir,
             args.max_work_dir_size,
         )
+
+    if args.container_runtime == "singularity":
+        singularity_folder = os.path.join(args.work_dir, 'codalab_singularity_images')
```

Comment:
Double check -- maybe check in deployment to see what work_dir is set to? to ensure it can store all these images?